### PR TITLE
Add Ubuntu support for test-kitchen intergration tests

### PR DESCRIPTION
**Pull request by patcon**
Needed to make some changes to the acme_server recipe, but should be pretty self explanatory :)

### CODE REVIEW

File: .kitchen.digitalocean.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+driver:
+  name: digitalocean
+  private_networking: false
+  size: 2gb
+
+transport:
+  name: rsync
```
Comment (patcon) [inline .kitchen.digitalocean.yml]: Speedier sync when working with a slow connection and getting the tests working.

File: test/fixtures/cookbooks/acme_server/recipes/default.rb [hunk 2]
```diff
@@ -87,16 +104,17 @@
...
-  command 'source /etc/profile.d/golang.sh && ./test/setup.sh 2>&1 && touch setup.done'
+  code 'source /etc/profile.d/golang.sh && ./test/setup.sh 2>&1 && touch setup.done'
   creates "#{boulderdir}/setup.done"
 end
 
 bash 'run_boulder' do
   cwd boulderdir
-  code 'source /etc/profile.d/golang.sh && /bin/screen -LdmS boulder ./start.py'
-  not_if '/bin/screen -list boulder | /bin/grep 1\ Socket\ in'
+  code 'source /etc/profile.d/golang.sh && screen -LdmS boulder ./start.py'
```
Comment (patcon) [inline test/fixtures/cookbooks/acme_server/recipes/default.rb]: `screen` command in different place on diff systems

File: test/fixtures/cookbooks/acme_server/recipes/default.rb [hunk 3]
```diff
@@ -87,16 +104,17 @@
   end
 end
 
-execute 'boulder_setup' do
+script 'boulder_setup' do
+  interpreter 'bash'
```
Comment (patcon) [inline test/fixtures/cookbooks/acme_server/recipes/default.rb]: needed to use script resource instead of execute in order to get "source" command

**Comment by thoutenbos** [comment]
Thanks!

**Pull request merged**